ClientApp/src/components/admin/dashboard-all/GiverTable.tsx

```diff
@@ -45,6 +60,25 @@ const Datatable: React.FC<Props> = ({ data, handleGiverChange, refreshData }) =>
     setOpenDialog(false);
   };
 
+  const saveComment = () => {
+    if (selectedGiver) {
+      apiservice
+        .post("admin/giver/addcomment", {
+          giverPartitionKey: selectedGiver.partitionKey,
+          giverRowKey: selectedGiver.rowKey,
```

Comment:
```suggestion
          event: selectedGiver.partitionKey,
          giverId: selectedGiver.rowKey,
```
When you rebase on dev after my PR, the selectedGiver object will have event and giverId instead of partitionKey and rowKey